frontend/integration-tests/tests/base.scenario.ts

```diff
@@ -27,6 +28,9 @@ describe('Create a test namespace', () => {
         .$('#confirm-action')
         .click();
       await browser.wait(until.urlContains(`/${testName}`), BROWSER_TIMEOUT);
+      execSync(
+        `oc label namespace ${testName} security.openshift.io/scc.podSecurityLabelSync=false pod-security.kubernetes.io/enforce=privileged --overwrite=true`,
```

Comment:
Based on https://kubernetes.io/docs/concepts/security/pod-security-standards/ `baseline` should be sufficient.
```suggestion
        `oc label namespace ${testName} security.openshift.io/scc.podSecurityLabelSync=false pod-security.kubernetes.io/enforce=baseline --overwrite=true`,
```